assets/js/functions.js

```diff
@@ -2,7 +2,7 @@
 const doc = document.documentElement;
 const inline = ":inline";
 // variables read from your hugo configuration
-const parentURL = '{{ absURL "" }}';
+const parentURL = '{{ absLangURL "" }}';
```

Comment:
Bug in #51 introduced in this PR /commit 😏, totally missed it.

Reply:
Oops, didn't realize it either :+1: